install/install.py

```diff
@@ -157,19 +157,23 @@ def install(self, cmd_arguments):
 
                 if fixsetuptools == 'y':
                     # backup the original symlink
-                    print("Backing up the original symlink...")
+                    Colorizer.normal("Backing up the original symlink...")
                     ts = time.time()
                     human_timestamp = datetime.fromtimestamp(ts).strftime('%Y-%m-%d-%H:%M:%S')
 
                     # backup the original symlink
-                    self.run_command("mv /usr/lib/python2.7/dist-packages/setuptools.egg-info /usr/lib/python2.7/dist-packages/setuptools.egg-info" + "-BACKUP-" + human_timestamp)
+                    self.run_command(
+                            "mv /usr/lib/python2.7/dist-packages/setuptools.egg-info /usr/lib/python2.7/dist-packages/setuptools.egg-info-BACKUP-" +
```

Comment:
The original path should be moved to a temporary variable. It would be:

+ Easier to update (only one variable to change, instead of two)
+ Make it possible to inline this and the `Colorizer.info` as well

Also, please use `%` or `format` to concatenate strings.